docs/overview/README.md

```diff
@@ -93,3 +122,94 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ``` 
+
+## Screenshots
+Below are the screenshots that highlight some of the features of the SkillTree platform. 
+These screenshots broken down into 2 sections:
+- [Dashboard](/overview/#screenshots): a web-based user-friendly interface that allows to create and manage gamified training profile.
+- [Ranking and Progress Display](/overview/#ranking-and-progress-display): JS libraries are integrated into your existing web-based application which provide your users a fully interactive user ranking and progress display.
+
+### Dashboard
+
+A Project is composed of Subjects which are made of Skills and a single skill defines a training unit within the
+gamification framework. Subjects are a way to group and organize skill definitions within a gameified training profile.
+Screenshots below demonstrates dashboard's page to create and manage subjects: 
```

Comment:
"Screenshots below demonstrate the creation and management of subjects in the dashboard"